core/jazz_ui/src/app/secondary-components/create-service/internal/create-service.component.html

```diff
@@ -483,7 +483,7 @@ <h4 class="bold-title">
                 </div>
             </section>
             <!-- start of aws events -->
-            <section class="each-step-wrap aws-events" *ngIf="typeOfService == 'function'">
+            <section class="each-step-wrap aws-events" *ngIf="typeOfService == 'function' && false">
```

Comment:
In internal it should be true though, no?

Reply:
Currently none of the files marked as internal do anything. Fixing that is part of the reverse pipeline story. This was just to keep them in sync.